frameworks/kafka/tests/test_jmx.py

```diff
@@ -0,0 +1,204 @@
+              "-s -v silent -n < jmx_domains_command.txt"
+
+        full_cmd = "bash -c '{}'".format(cmd)
+
+        rc, output, stderr = sdk_cmd.run_cli("task exec {} {}".format(broker_task_id_0, full_cmd), print_output=True)
+        print(rc)
+        print(output)
+        print(stderr)
+
+        assert "kafka.server" in output
+        assert "kafka.controller" in output
+
+    finally:
+        sdk_install.uninstall(config.PACKAGE_NAME, foldered_name)
+        uninstall_jmx_secrets()
+
+
+@pytest.mark.sanity
+def test_secure_jmx_dcos_crt():
+    foldered_name = sdk_utils.get_foldered_name(config.SERVICE_NAME)
+
+    try:
+        install_jmx_configured_kafka(self_signed_trust_store=False)
+        broker_task_id_0 = sdk_tasks.get_task_ids(foldered_name)[0]
+        install_jmxterm(task_id=broker_task_id_0)
+        generate_jmx_command_files(task_id=broker_task_id_0)
+
+        cmd = "export JAVA_HOME=$(ls -d ${MESOS_SANDBOX}/jdk*/jre) && " \
+              "${JAVA_HOME}/bin/java " \
+              "-Duser.home=${MESOS_SANDBOX} " \
+              "-Djdk.tls.client.protocols=TLSv1.2 -Djavax.net.ssl.trustStore=${JAVA_HOME}/lib/security/cacerts " \
+              "-Djavax.net.ssl.trustStorePassword=changeit " \
+              "-Djavax.net.ssl.keyStore=$MESOS_SANDBOX/jmx/key_store -Djavax.net.ssl.keyStorePassword=deleteme "  \
+              "-Djavax.net.ssl.trustStoreType=JKS -Djavax.net.ssl.keyStoreType=JKS -jar jmxterm-1.0.1-uber.jar " \
+              "-l service:jmx:rmi:///jndi/rmi://${MESOS_CONTAINER_IP}:31199/jmxrmi -u admin -p adminpassword " \
+              "-s -v silent -n < jmx_beans_command.txt"
+
+        full_cmd = "bash -c '{}'".format(cmd)
+
+        _, output, _ = sdk_cmd.run_cli("task exec {} {}".format(broker_task_id_0, full_cmd), print_output=True)
+
+        assert "kafka.server:type=kafka-metrics-count" in output
+        assert "kafka.server:name=BrokerState,type=KafkaServer" in output
+
+    finally:
+        sdk_install.uninstall(config.PACKAGE_NAME, foldered_name)
+        uninstall_jmx_secrets()
+
+
+def random_string(length=10):
+    letters = string.ascii_lowercase
+    return ''.join(random.choice(letters) for i in range(length))
+
+
+def write_to_file(content, file_path):
+    text_file = open(file_path, "w+")
+    text_file.write(content)
+    text_file.close()
+
+
+def generate_jmx_command_files(task_id: string):
+    cmd = "\n".join(
+        [
+            "echo beans >> jmx_beans_command.txt && ",
+            "echo domains >> jmx_domains_command.txt",
+        ]
+    )
+    full_cmd = "bash -c '{}'".format(cmd)
+    rc, _, _ = sdk_cmd.run_cli("task exec {} {}".format(task_id, full_cmd), print_output=True)
+    assert rc == 0, "Error creating jmx_commands file"
+
+
+def install_jmxterm(task_id: string):
+    jmx_term_url = 'https://github.com/jiaqi/jmxterm/releases/download/v1.0.1/jmxterm-1.0.1-uber.jar'
```

Comment:
If there aren't any licensing concerns, I would suggest moving this jar to our `downloads.mesosphere` bucket to safeguard against this user ever deleting the file

Reply:
There are not licensing issues, will upload now

Reply:
using now https://downloads.mesosphere.io/jmx/assets/jmxterm-1.0.1-uber.jar